rebasehelper/tests/conftest.py

```diff
@@ -38,3 +38,15 @@ def workdir(request, tmpdir_factory):
         for file_name in getattr(request.cls, 'TEST_FILES', []):
             shutil.copy(os.path.join(TEST_FILES_DIR, file_name), wd)
         yield wd
+
+
+def pytest_collection_modifyitems(items):
+    for item in items:
+        # item is an object of Function class.
```

Comment:
I'd use `instance of Function class`, is sounds better to me.

Reply:
OK I will change for that.